### PR TITLE
UI automation refactor mk1: Using `trackTest()` and `trackStep()` in older tests

### PR DESCRIPTION
**Pull request by pachlava**
The only affected part of functionality is more logging.

Note: I'm not considering this logging approach as final or most suitable, just decided to apply it to older tests since it's already in place, and required minimal time investments.

### Fix
- [x] Inserting `trackTest()` and `trackStep()` to older tests for better logging
- [x] Renamed `NotesList.swift` ~> `NoteList.swift`
- [x] Removed several obsolete commented out lines from tests

### Test
1. Running all tests, seeing that there's a simplistic log for every test, can be filtered by using `>` for output filter:

Screenshot 2021-03-04 at 15 59 15

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.

### CODE REVIEW

**Review by mokagio — APPROVED** [review]
> Note: I'm not considering this logging approach as final or most suitable, just decided to apply it to older tests since it's already in place, and required minimal time investments.

👍

I recommend looking into [Apple's way of doing this](https://developer.apple.com/documentation/xctest/activities_and_attachments/grouping_tests_into_substeps_with_activities).

**Comment by mokagio** [comment]
The reason the tests failed on CI, and will keep failing even if you try to run them again, has to do with how the pipeline is configured in regards to checking out the Git repo.

I'm guessing what happened is that when the branch for #1190 was pushed, the build for this one had just started but then the commit wasn't available to the shallow commit anymore.

Given #1190 points to this branch, once merged a new commit will land here and a new build will start which should succeed and unlock the merge button.

Let me know if you'd rather merge earlier and we can look at how to do it.

**Comment by pachlava** [comment]
Thanks @mokagio!

> Given #1190 points to this branch, once merged a new commit will land here and a new build will start which should succeed and unlock the merge button.

This is exactly what has happened just now

**Pull request merged**
